config/ingest-gen3.py

```diff
@@ -1,2 +1,2 @@
-config.padRegionAmount = 4000
-config.instrument = "lsst.obs.lsst.gen3.LatissInstrument"
+config.padRegionAmount = 4000  # noqa : F821
+config.instrument = "lsst.obs.lsst.gen3.LatissInstrument"  # noqa : F821
```

Comment:
I don't think this is needed -- butler ingest works out the instrument for itself and you should be able to add ComCam data to the same butler repo when the time comes.